jinahub/rankers/MinRanker/minranker.py

```diff
@@ -22,11 +22,16 @@ def __init__(
         self,
         metric: str,
         default_traversal_paths: Iterable[str] = ('r',),
+        metric_is_distance: bool = False,
         *args,
         **kwargs
     ):
         super().__init__(*args, **kwargs)
         self.metric = metric
+        if metric_is_distance:
+            self.distance_mult = 1
+        else:
+            self.distance_mult = -1
```

Comment:
What is the motivation to add this part? I think the user should be aware of this when they choose the index. Therefore this should be done in the indexer part.

Reply:
> Therefore this should be done in the indexer part.

@nan-wang Does that mean the matched score resulted from indexer should always be a distance (or similarity) value? I don't think we can guarantee this assumption.

Reply:
I was a bit worried here because we actually assume that all metrics are distance metrics. We can not use scores (the higher the better) at all currently.

Reply:
If this is a design choice, that is also fine for me. Then I only expect distance metrics here :smile:

Reply:
If things are a distance metric, we need to switch the sign in the sorting operations, however.